mechanics/TOC.md

```diff
@@ -0,0 +1,92 @@
+employees of the same vendor.
+
+There will be an annual election to determine the composition of the TOC for the
+following year. Three seats will be up for election in one year and two will be
+up for election the following year.
+
+# Candidate Eligibility
+
+Current TOC members and
+[Approvers](https://github.com/knative/community/blob/master/ROLES.md#approver)
+with at least 3 months tenure are eligible to stand for election. Candidates may
+self-nominate or be nominated by another eligible member. The approximate time
+commitment of a TOC member is around 8 hours per week.
+
+# Voter Eligibility
+
+Anyone who has at least 50 contributions in the last 12 months is eligible to
+vote in the TOC election. Contributions are defined as opening PRs, reviewing
+and commenting on PRs, opening and commenting on issues, writing design docs,
+commenting on design docs, helping people on slack, participating in working
+groups and etc.
+
+[This
+dashboard](https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year)
+shows only GitHub based contributions and does not capture all the contributions
+we value. *We expect this metric not to capture everyone who should be eligible
+to vote.* If a community member has had significant contributions over the past
+year but is not captured in the stats.knative.dev dashboard, they will be able
+to submit an exception form to the steering committee who will then review and
+determine whether this member should be marked as an exception.
+
+All eligible voters will be captured at
+`github.com/knative/community/elections/$YEAR/voters.md` and the voters’ guide
+will be captured at `github.com/knative/community/elections/$YEAR/README.md`
+similar to the kubernetes election process.
+
+We are committed to an inclusive process and will adapt future eligibility
+requirements based on community feedback.
+
+# Election Process
+
+Elections will be held using a time-limited
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+[CIVS](http://civs.cs.cornell.edu/) using the
+[Schulze](https://en.wikipedia.org/wiki/Schulze_method) method. The top
+vote-getters will be elected to the open seats. This is the same process used by
+the Kubernetes project.
+
+## Election Officers
+
+The steering committee will be the election officers for the technical oversight
+committee elections.
+
+## Vacancies
+
+In the event of a resignation or other loss of an elected governance board
+member, the candidate with the next most votes from the previous election will
+be offered the seat. This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible. Eligible voters from the most recent election will
+vote in the special election (ie: eligibility will not be redetermined at the
+time of the special election). A board member elected in a special election will
```

Comment:
I think this should state that *any* replacement "committee" member serves out the term this way, not just those elected in a special election.